### PR TITLE
CIV-3756 change site id for specified claim for service payment request

### PR DESCRIPTION
**Pull request by deepthidoppalapudihmcts**
Before creating a pull request make sure that:

 commit messages are meaningful and follow good commit message guidelines
 README and other documentation has been updated / added (if needed)
 tests have been updated / new tests has been added (if needed)
 Screenshot attached for the Evidence of manual testing
 Functional, Smoke and API tests have been run locally
 Full gradle build and run tests with coverage have been completed (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-3756

### Change description ###

For a specified claim we should be setting the site id as AAA6 as per civil service make pba payment logic. We should put site id for spec in the application yaml and use that value to be passed when making a service request call.  We should use generalAppSuperClaimType to decide if its spec or unspec then choose the site id accordingly and update it in this pojo while creating a service request. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```
functional_test
smokeTest
api Test

### CODE REVIEW

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.civil.enums;
```
Comment (smitatalawadekar-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java:1]: where is this class being used? I see its usage only as part of tests... we should not be introducing models only for test purpose.

**Comment by karthick-mohanasundaram-HMCTS** [comment]
@deepthidoppalapudihmcts  could you please add the screenshot of grawdle build?

File: src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java (hunk 1 above)
Comment (deepthidoppalapudihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java:1]: we can use this later when we refactor code for spec and unspec claim.,

**Comment by deepthidoppalapudihmcts** [comment]
image

File: src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java (hunk 1 above)
Comment (smitatalawadekar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java:1]: We can introduce it later too when we need it.

Comment (deepthidoppalapudihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java:1]: I have removed safely

**Review by Azam-Hmcts — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

**Pull request merged**